src/App.js

```diff
@@ -14,11 +15,111 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+function removeSpaces(value) {
+  return value.split(' ').join('');
+}
+
+class App extends React.Component {
+  state = {
+    goods: [],
+    elemIsVisible: false,
+  }
+
+  showList = () => {
```

Comment:
```suggestion
  handleStart = () => {
```
https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364